ports/atmel-samd/boards/metro_m0_express/mpconfigboard.mk

```diff
@@ -12,8 +12,25 @@ EXTERNAL_FLASH_DEVICES = "S25FL216K, GD25Q16C"
 LONGINT_IMPL = MPZ
 
 CIRCUITPY_BITBANGIO = 0
+CIRCUITPY_COUNTIO = 0
+CIRCUITPY_GAMEPAD = 0
```

Comment:
AUDIOBUSIO is removed from (nearly?) all the other SAMD21 Express builds except for Metro M0. Could we be consistent about what's in generic M0 Express builds? Here instead GAMEPAD was removed, but you left I2CPERIPHERAL.

If we can be consistent, then these defaults could be moved to mpconfigport.h in the SAMD21 conditional section.